draft-ietf-quic-http.md

```diff
@@ -1902,7 +1903,7 @@ The entries in the following table are registered by this document.
 | Name                                | Code       | Description                              | Specification          |
 | ----------------------------------- | ---------- | ---------------------------------------- | ---------------------- |
 | HTTP_NO_ERROR                       | 0x0000     | No error                                 | {{http-error-codes}}   |
-| HTTP_WRONG_SETTING_DIRECTION        | 0x0001     | Setting sent in wrong direction          | {{http-error-codes}}   |
+| HTTP_SETTINGS_ERROR                 | 0x0001     | Setting duplication or wrong direction   | {{http-error-codes}}   |
```

Comment:
I'd be more general -- SETTINGS frame contained invalid values, or something like that.  It's anything that could go wrong with that frame's payload.

Reply:
Thanks, fixed.